helm_deploy/values-dev.yaml

```diff
@@ -1,7 +1,7 @@
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
 
-replicaCount: 0
+replicaCount: 1
```

Comment:
should really be 2 but happy for a new PR to correct that once it is up and running

Reply:
easy enough to fix along with the other changes